### ISSUE TITLE
Cordova doesn't link to correct files

### ISSUE THREAD

**Issue by jakobdamjensen** [TAPevents/tap-i18n#25]
The translation files doesn't seem to get through when doing a cordova build. Have you tried this?

**Comment by Kestanous** [comment]
:+1: Going with the one language rename to en for now.

**Comment by Kestanous** [comment]
Any updates or timeline on this? What is the issue, any pointers for looking into a PR?

**Comment by slickdev** [comment]
+1

**Comment by theosp** [comment]
Support for Cordova when internet connection is available added in v1.4.0.

https://github.com/TAPevents/tap-i18n#using-tap-i18n-in-cordova-apps

Thanks to @slickdev for funding it.

**Comment by nerdmed** [comment]
Hey @theosp, i am currently working on a cordova project and i think its critical to have i18n on mobile devices without requiring internet connection. As the language files are bundled by meteor to the application it should be possible to load them completely without internet connection.  I would try to create a PR for this if you would take the time to describe your current solution and the problem, this would save me a lot of time.  

Best,
Ahmed

**Comment by theosp** [comment]
Hi Ahmed,

Work in this direction is currently in progress for v1.6 , will post it the
release candidate as it'll be ready so you can help with testing.

-Daniel

**Comment by arathipanya** [comment]
Hi, I am now using v1.7 and I have the same issue when trying to run my app on an iOS device: "Couldn't load language 'fr' JSON: error"
On desktop it works fine. Do you have any idea?

**Comment by arjank** [comment]
I have a similar problem with Meteor 1.2.1 and tap-i18n 1.7.0. Translations are not loaded on Android nor on iOS, but the app itself is working. Translations are working on my desktop development environment.

When testing on Android, I get this error message:
```
(android:http://meteor.local/:0) XMLHttpRequest cannot load http://192.168.2.97:3000/i18n/de.json. No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'http://meteor.local' is therefore not allowed access.
```

Any idea how to resolve this issue?

**Comment by miri-am** [comment]
@arjank @arathipanya. Did you find a solution for this?

**Comment by arathipanya** [comment]
@miri-am Unfortunately no... At last I just created a simple cordova project linked to my server, where my Meteor app is hosted.

**Comment by arjank** [comment]
@miri-am not really. It seems to work when I remove the app from the phone and then re-build / re-install it, so it could be a caching issue.

**Comment by inspiraluna** [comment]
@arjank using meteor 1.2.1, I realized if you test locally don't use the mobile-server option it confuses the loading imho. And yes deleting the app or deleting all app data helps often. 

Also I changed my cdn_path in project-tab.18n to the url on production server with this https://www.productionserver.com/i18n since my app did not even continued to load.  It could not connect to the right cdn_path. it was looking on a completely wrong server. an ip which is still a miracle to me.
